grails-app/views/layouts/_heatmap.gsp

```diff
@@ -6,18 +6,39 @@
         %{-- Fetch Data --}%
         %{--========================================================================================================--}%
         <workflow-tab tab-name="Fetch Data">
-            <concept-box
-                concept-group="fetch.conceptBoxes.highDimensional"
-                type="HD"
-                min="1"
-                max="512"
-                label="High Dimensional"
-                tooltip="Select high dimensional data node(s) from the Data Set Explorer Tree and drag it into the box. The nodes needs to be from the same platform.">
-            </concept-box>
-
-            %{--TODO include low dimensions--}%
-            %{--<concept-box concept-group="fetch.conceptBoxes.numerical"></concept-box>--}%
-            %{--<concept-box concept-group="fetch.conceptBoxes.categorical"></concept-box>--}%
+
+            <table>
+                <tr>
+                    <td>
+                        <concept-box concept-group="fetch.conceptBoxes.highDimensional"
+                                     type="HD"
+                                     min="1"
+                                     max="-1"
+                                     label="High Dimensional Variables"
+                                     tooltip="Select high dimensional data node(s) from the Data Set Explorer Tree and drag it into the box. The nodes needs to be from the same platform.">
+                        </concept-box>
+                    </td>
+                    %{--<td>--}%
```

Comment:
Is there a reason this commented-out code is still here? If it is not used for anything, I recommend deleting it.